app/controllers/students_controller.rb

```diff
@@ -1,6 +1,7 @@
 # Controller for Students
 class StudentsController < ApplicationController
```

Comment:
🎱  Might be worth following common JSON API namespace like the snippet below so versioning is set and the API is decoupled:

```rb
module Api
  module V1
    class StudentsController < ApplicationController
      # ... code here
    end
  end
end
```
I imagine your API won't be used by third parties, but it's still nice to do so you can build up the habit.

Reply:
This is the one part I don't totally understand. I'll revisit it once I've finished these CRUD operations and have done some research.